### ISSUE TITLE
[Feature Request] List Resistances and Vulnerbilities

### ISSUE THREAD

**Issue by yodaldevoid** [MalukuSeito/Character-Builder-5#8]
Unless I've missed it (and I absolutely could have) there is no place where resistances and vulnerabilities are listed on a character sheet.

**Comment by MalukuSeito** [comment]
That is true, I need to find or make a sheet that has a space for them, haven't gotten around to it yet. They are available as Resistances/Vulnerabilites/... PDFFields if you want to modify the DefaultPdf.xml/AlternatePDF.xml files.

I plan on reworking the PDF Export a bit anyway.
